src/Controller/OrderItemController.php

```diff
@@ -98,16 +98,12 @@ public function addProductBundleAction(Request $request): Response
             return $this->handleBadAjaxRequestView($configuration, $form);
         }
 
-        $view = View::create()
-            ->setData([
-                'configuration' => $configuration,
-                $this->metadata->getName() => $orderItem,
-                'form' => $form->createView(),
-            ])
-            ->setTemplate($configuration->getTemplate(CartActions::ADD . '.html'))
-        ;
-
-        return $this->viewHandler->handle($configuration, $view);
+        return $this->render(
+            $configuration->getTemplate(CartActions::ADD . '.html'),
+            ['configuration' => $configuration,
+            $this->metadata->getName() => $orderItem,
+            'form' => $form->createView(),]
+        );
```

Comment:
Fix lint like e.g.
```php
        return $this->render(
            $configuration->getTemplate(CartActions::ADD . '.html'),
            [
                'configuration' => $configuration,
                $this->metadata->getName() => $orderItem,
                'form' => $form->createView(),
            ]
        );
```
You can also use shortcut `Ctrl + Alt + L` when using PhpStorm